Doc/library/datetime.rst

```diff
@@ -2411,8 +2410,8 @@ requires, and these work on all platforms with a standard C implementation.
 |           |                                |  Samstag (de_DE)       |       |
 +-----------+--------------------------------+------------------------+-------+
 | ``%w``    | Weekday as a decimal number,   | 0, 1, ..., 6           |       |
-|           | where 0 is Sunday and 6 is     |                        |       |
-|           | Saturday.                      |                        |       |
+|           | where 0 is Sunday              |                        |       |
+|           | and 6 is Saturday.             |                        |       |
```

Comment:
What is changed?

Reply:
This remark is outdated; marking it as resolved.